ping/rust/Dockerfile

```
@@ -14,7 +14,7 @@ RUN cd ./plan/ && \
         cargo update --dry-run && \
         cargo fetch && \
         # use a default feature to rely on docker caching.
-        cargo build --release --features "libp2pv0450"
+        cargo build --release --features="${CARGO_FEATURES}"
```

```suggestion
        cargo build --release --features="libp2pv0480"
```

How about we bump the version to a more recent one instead of taking the second to oldest one? My hope would be an increase in overlap. Just an idea. Feel free to close @laurentsenta.